ticket.go

```diff
@@ -266,3 +275,27 @@ func (s *TicketService) GetOne(id string) (*Ticket, *Response, error) {
 
 	return ticket, resp, err
 }
+
+// Create a new Zendesk Ticket
+func (s *TicketService) Create(ticket *Ticket) (*Ticket, *Response, error) {
+	url := "tickets.json"
+
+	body, err := json.Marshal(&ticket)
+
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req, err := s.client.NewRequest("POST", url, body)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	response := TicketResponse{}
```

Comment:
Is it really returning a TicketResponse object? Because that looks like it is more suited for a collection of Tickets not a single one. You may have to create a new struct to accommodate a single ticket response. It would probably make sense to call that struct `TicketResponse` and rename the existing one to `TicketCollectionResponse`.